cmd/sriov-network-config-daemon/service.go

```diff
@@ -152,6 +152,19 @@ func phasePre(setupLog logr.Logger, conf *systemd.SriovConfig, hostHelpers helpe
 	hostHelpers.TryEnableTun()
 	hostHelpers.TryEnableVhostNet()
 
+	if conf.RdmaMode != "" {
+		rdmaSubsystem, err := hostHelpers.GetRDMASubsystem()
+		if err != nil {
+			setupLog.Error(err, "failed to get RDMA subsystem mode")
```

Comment:
If conf.RdmaMode is not empty string, then the user explicitly requested RDMA mode configuration. I think we can return error in  this case. WDYT?

Reply:
done